src/components/Markdown/Hr.tsx

```diff
@@ -1,6 +1,6 @@
-import { Divider, DividerProps } from "@chakra-ui/react";
+import { DividerProps } from "@chakra-ui/react";
 import type { FunctionComponent } from "react";
 
-export const Hr: FunctionComponent<DividerProps> = (props) => (
-  <Divider my={10} {...props} />
-);
+// HRs are abused in API reference docs so a temporary hack is to simply filter
+// them out until we remove them from the generated docs.
+export const Hr: FunctionComponent<DividerProps> = () => <></>;
```

Comment:
As a warning - this will remove any dividers that users put in their readme files.